testing-guidance/workflow-tests/ragdoll/ragdoll-authoring-improvements.md

```diff
@@ -0,0 +1,47 @@
+| **Manipulators** | <ol><li>In the Animation Editor open the Action History, by going to View → Action History. Place the window in a visible, and convenient place (for example dock it under the Skeleton Outliner pane).</li> <li>Modify any bone using any of the manipulators in the Atom Renderer Window (same as in the "Verify Atom Render Window UI clusters" workflow). Note if the relevant properties are adjusted in the Ragdoll pane, and an action is registered to the Action History window.</li> <li>Modify any bone by changing properties related to the button clusters under the Ragdoll pane (for example "Swing limit Y", or "Twist upper limit"). Note if the relevant properties are adjusted in the Ragdoll pane, and an action is registered to the Action History window.</li> <li>Perform Undo/Redo actions on both the changes made by using Atom Render Window, and modifying properties under the Ragdoll pane. Undo/Redo actions can be performed by using Ctrl+Z/Ctrl+Shift+Z shortcuts, or by selecting Edit → Undo/Edit → Redo.</li></ol> | <ul><li>Action History can be opened.</li><li>Modifications made in the Atom Render Window using the manipulators is saved to the Action History, and is affected by Undo/Redo actions.</li><li>Modifications made in the Ragdoll pane by adjusting properties is saved to the Action History, and is affected by Undo/Redo actions.</li></ul>  |
```

Comment:
Step 2: Move the Note/Validation to the Things to Watch Out For.
Step 3: Move the Note/Validation to the Things to Watch Out For
Step 4: Move the Note/Validation to Things to Watch Out For

Reply:
Adjusted steps 2, and 3. I was not sure on what to move from the step 4, as there is no note/validation listed in it so I just fixed an error in the text, and left it as is otherwise. Please let me know if I am overlooking something in step 4 after the second commit goes through.